docs/cunumeric/source/user/usage.rst

```diff
@@ -0,0 +1,96 @@
+Usage
+=====
+
+Drop-in replacement
+-------------------
+
+Using cuNumeric as a replacement for NumPy is easy. Users only need
+to replace:
+
+.. code-block:: python
+
+  import numpy as np
+
+with
+
+.. code-block:: python
+
+  import cunumeric as np
+
+Running cuNumeric programs
+--------------------------
+
+Using ``legate``
+~~~~~~~~~~~~~~~~
+
+The preferred way to run cuNumeric programs is with the ``legate`` driver:
+
+.. code-block:: sh
+
+  legate --cpus 2 --gpus 2 script.py <script options>
+
+All ``legate`` command line configuration options must come *before* the script
+that is be executed. Any options after the script are passed on to the script.
+
+It is also possible to pass the ``legate`` command line configuration options
+using the ``LEGATE_CONFIG`` environment variable:
+
+.. code-block:: sh
+
+  LEGATE_CONFIG="--cpus 2 --gpus 2" legate script.py <script options>
+
```

Comment:
Note here that LEGATE_CONFIG has lower precedence.

Reply:
Or maybe note that in configuration.rst, and link that page from here ("see configuration.rst for more details on configuration options").

Reply:
> Note here that LEGATE_CONFIG has lower precedence.

just checking,  you mean lower than command line options?

Reply:
Right

Reply:
https://github.com/nv-legate/cunumeric/pull/805/commits/111bc9affb4f10755b446997fbd2126c738f2ce0